src/pages/OperationsPage.js

```diff
@@ -47,10 +48,12 @@ class OperationsPage extends Component {
     }
 
     componentWillMount(){
-        this.props.getConfirmationsOnce();
-        this.props.getListCBE();
-        this.props.getPendingsOnce();
-        this.props.getLOCsOnce();
+        this.props.getConfirmationsOnce();   //  todo
+        this.props.getListCBE();   //  todo
+        this.props.getPendingsOnce();   //  todo
+        if (this.props.isLOCsNeedReload) {
```

Comment:
@Pavel-DV, we are fetching/loading a lot of things many times for each abstracted state. Actually now we need two common event flags in most of the components - isFetching & isReady (now called just 'ready'). We can rename 'ready', 'isLOCsNeedReload' and another such flags to 'isReady'. This name will be clear when developer will see at list one implementation of this event in any component.